ztp/kube-compare-reference/compare.sh

```diff
@@ -0,0 +1,41 @@
+#! /bin/bash
+
+function read_dir() {
+for file in `ls $1`
+  do 
+    if [ -d $1"/"$file ]
+    then
+      read_dir $1"/"$file
+    else
+      echo $1"/"$file
+    fi
+done
+}
+
+
+read_dir renderedv1 |grep yaml  > rendered_file
+read_dir ../source-crs |grep yaml  > source_file
+
+for file1 in `cat source_file`
+do
+    for file2 in `cat rendered_file`
+    do
+      if [ "${file1##*/}" = "${file2##*/}" ]
+      then
+        diff -u $file1 $file2
+        printf "\n"
+        echo $file1 >> same_file
```

Comment:
Instead of a temporary file, you could use a bash array for this if you like

Reply:
It's difficult to filter out the same file name from source_file by using array because of different prefix.
I can try that approach in later release.

Reply:
That's fine!  If a temporary file is easier, stick with it!